url/url.cpp

```diff
@@ -191,6 +191,74 @@ std::string blob_url_create(Origin const &origin) {
     return result;
 }
 
+// https://url.spec.whatwg.org/#concept-host-serializer
+std::string Host::serialize() const {
+    if (type == HostType::Ip4Addr) {
+        return util::ipv4_serialize(std::get<std::uint32_t>(data));
+    } else if (type == HostType::Ip6Addr) {
+        return util::ipv6_serialize(const_cast<std::array<std::uint16_t, 8> &>(std::get<2>(data)));
```

Comment:
`Otherwise, if host is an IPv6 address, return U+005B ([), followed by the result of running the IPv6 serializer on host, followed by U+005D (]).`

Missing the `[]`s.